tests/generator/generator.cpp

```diff
@@ -102,7 +102,7 @@ TEST(Generator, twoVariablesOfIntegration)
     EXPECT_EQ(size_t(0), parser->issueCount());
 
     const std::vector<std::string> expectedIsues = {
-        "Variable 'time' in component 'main' of model 'two_variables_of_integration' and variable 'other_time' in component 'sub_sub_sub' of model 'two_variables_of_integration' cannot both be a variable of integration.",
+        "Variable 'time' in component 'main' and variable 'other_time' in component 'sub_sub_sub' cannot both be a variable of integration.",
```

Comment:
Can CellML handle only one variable of integration at a time in the entire model (in which case maybe the error message should be "**the** variable of integration", or is it that these are connected variables, in which case the error message should say that?

Reply:
In principle, CellML supports more than one variable of integration (at least in CellML 1.0/1.1, I can't recall for certain in CellML 2.0...!?), but @nickerso and I once had a chat about this and we agreed that we should only support one variable of integration when it comes to code generation.

So, you are correct, it should "**the** variable of integration" and not "**a** variable of integration". Done in commit 04445ba3.